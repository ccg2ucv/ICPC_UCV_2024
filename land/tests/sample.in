0 0 4 0 4 4 0 4
2 2 6 2 6 6 2 6
