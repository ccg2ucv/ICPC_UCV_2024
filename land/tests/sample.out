YES 9 20 25


